sale_order_import/wizard/import_sale.py

```diff
@@ -0,0 +1,737 @@
+        "account.journal",
+        string="Customer Invoice Journal",
+        required=True,
+        default=_default_customer_invoice_journal,
+    )
+    customer_payment_journal_id = fields.Many2one(
+        "account.journal",
+        string="Customer Payment Journal",
+        default=_default_customer_payment_journal,
+    )
+    asynchronous = fields.Boolean(string="Process import asynchronously", default=True)
+    process_payment = fields.Boolean("Process Payments")
+
+    @api.model
+    def _get_order_value_dict(
+        self,
+        error_vals,
+        partner_tel,
+        product_id,
+        pricelist_id,
+        warehouse_id,
+        team_id,
+        carrier_id,
+        partner_dict,
+        product_dict,
+        pricelist_dict,
+        picking_dict,
+        warehouse_dict,
+        team_dict,
+        carrier_dict,
+    ):
+        """Get order value dict"""
+        partner_value = partner_tel
+        ctx = self._context.copy()
+
+        partner_name_value = False
+        if self._context.get("partner_name", False):
+            partner_name_value = self._context["partner_name"].strip()
+            ctx.update({"partner_name": partner_name_value})
+        self.with_context(**ctx)._get_partner_dict(partner_value, partner_dict)
+
+        product_id_value = product_id
+        self._get_product_dict(product_id_value, product_dict, error_vals)
+
+        pricelist_value = pricelist_id
+        self._get_pricelist_dict(pricelist_value, pricelist_dict, error_vals)
+
+        warehouse_value = warehouse_id
+        self._get_picking_dict(
+            warehouse_value, picking_dict, warehouse_dict, error_vals
+        )
+
+        team_value = team_id
+        self._get_team_dict(team_value, team_dict, error_vals)
+
+        carrier_value = carrier_id
+        if carrier_value:
+            self._get_carrier_dict(carrier_value, carrier_dict, error_vals)
+
+        return (
+            partner_value,
+            product_id_value,
+            pricelist_value,
+            warehouse_value,
+            team_value,
+            carrier_value,
+        )
+
+    @api.model
+    def _get_order_value(self, error_vals, taxes, price_unit, taxes_id, product_qty):
+        """Get order value"""
+        tax_from_chunk = taxes_id
+        if tax_from_chunk:
+            self._get_taxes(tax_from_chunk, taxes, error_vals)
+
+        qty = product_qty
+        try:
+            qty = float(qty)
+            if qty <= 0:
+                error_vals["error_message"] = (
+                    error_vals["error_message"]
+                    + _('Column "Line Qty" must be greater than 0.')
+                    + "\n"
+                )
+                error_vals["error"] = True
+        except ValueError:
+            error_vals["error_message"] = (
+                error_vals["error_message"]
+                + _('Column "Line Qty" must be a number.')
+                + "\n"
+            )
+            error_vals["error"] = True
```

Comment:
Why do we still have this check here when it should be checked in `_check_field_vals()`?